src/bus/policy.c

```diff
@@ -13,6 +13,12 @@
 #include "util/error.h"
 #include "util/selinux.h"
 
+enum {
+        CONFIG_TRISTATE_UNSET,
+        CONFIG_TRISTATE_YES,
+        CONFIG_TRISTATE_NO,
```

Comment:
CONFIG? That should by `POLICY_TRISTATE_*` shouldn't it?

Reply:
So I dislike that we treat this as a common definition. We have this type-definition in the launcher, and now here as well, and we silently assume that the wire uses the same definition. Can we get some common solution here?

I marked the two places below where we read this type from the variant without any type-conversion (since both types match right now). Shouldn't there be some check that the value is not higher than 2?

I would be fine with placing this in src/util/common.h and naming it something like `UTIL_TRISTATE_*`.

Reply:
Agreed. Changed.